Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5046.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if APP
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 5046, "CornerRadius doesn't work in explicit style when implicit style exists", 
+	       PlatformAffected.Android | PlatformAffected.iOS)]
```

Comment:
No UWP platform? Since that was the original culprit

Reply:
I could only test it on Android and iOS, but you're right the original issue was UWP and I guess this should also happen there - I added PlatformAffected.UWP

Reply:
No problem, this way the UWP tests on our side will trigger as well so they will run for you there. Whenever it runs into a problem we'll let you know. Or, if you are not able to fix it since you do not have Windows, we will have a look